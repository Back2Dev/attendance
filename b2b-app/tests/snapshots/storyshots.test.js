/**
 * Runs StoryShots tests.
 * Snapshot testing of all Storybook Stories.
 */
import initStoryshots from '@storybook/addon-storyshots'
import { render } from '@testing-library/react'

const reactTestingLibrarySerializer = {
  print: (val, serialize) => serialize(val.container.firstChild),
  /* eslint-disable-next-line no-prototype-builtins */
  test: (val) => val && val.hasOwnProperty('container'),
}

initStoryshots({
  renderer: render,
  snapshotSerializers: [reactTestingLibrarySerializer],
})
