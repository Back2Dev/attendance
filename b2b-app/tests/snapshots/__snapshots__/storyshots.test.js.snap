// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Admin/Register/Form/Field Grid Default Items 1`] = `
<div
  style="border: 1px solid lightgrey; padding: 1rem; margin: 3rem auto; max-width: 600px;"
>
  <div
    class="makeStyles-root-1 makeStyles-root-2"
  >
    <div
      style="grid-column-end: span undefined;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="Name"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      style="grid-column-end: span undefined;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="Phone"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Admin/Register/Form/Field Grid Multi Span Items 1`] = `
<div
  style="border: 1px solid lightgrey; padding: 1rem; margin: 3rem auto; max-width: 600px;"
>
  <div
    class="makeStyles-root-5 makeStyles-root-6"
  >
    <div
      style="grid-column-end: span 2;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="First Name"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      style="grid-column-end: span 2;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="Last Name"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      style="grid-column-end: span 4;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="Address"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      style="grid-column-end: span 2;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="Suburb"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      style="grid-column-end: span 1;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="State"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      style="grid-column-end: span 1;"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth"
      >
        <input
          class="MuiInputBase-input MuiInput-input"
          placeholder="Postcode"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
